swagger/v1/swagger.yaml

```diff
@@ -0,0 +1,124 @@
+---
+openapi: 3.0.1
+info:
+  title: The Open Food Network
+  description: |-
+    This spec is auto generated using the rswag gem. It is incomplete and not yet valid for openapi 3.0.1. Do not publish this. \
```

Comment:
"Do not publish this."
It's ok if you think this is the best thing to do now.
I feel we can just move forward as much as we can already. I'd just keep the engine and have this public. We can then improve it.

Reply:
For now our API documentation 'process' is to update the swagger.yaml in root and then update https://app.swaggerhub.com/. I think we should only add things to the repo that are in use, not that may be used later, no matter how certain.

For me that means not adding the engine, as it's not in use yet. 

Rswag on the other hand is being used as the API spec is active, and I don't mind the half baked new swagger.yaml being committed either, as we'll now be able to see progress on it as more tests are added, and rswag issues are fixed. So arguably that is in use -- it serves a purpose.

Reply:
ah, ok, let's see if we are on the same page here.

"For now our API documentation 'process' is to update the swagger.yaml in root and then update "

I thought from now on the api doc process is to write a spec and rerun rswag? Is it not?
I thought the engine would publish the latest version of that doc, which is incomplete. The manual swagger doc is also incomplete.

Reply:
I think I am saying I'd be happy to deprecate the manual process already. To add context, it has been used at all, it was created by a group of students and updated by me once.

Reply:
I don't have a strong opinion about this! I am just trying to clarify where we are going with this :+1:

Reply:
Actually, never mind, from reading your todo list above I understand what's your plan :+1: